core/platform/lf_linux_support.c

```diff
@@ -33,11 +33,14 @@ THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "platform.h"
 
 #if defined NUMBER_OF_WORKERS || defined LINGUA_FRANCA_TRACE
-#if __STDC_VERSION__ < 201112L || defined (__STDC_NO_THREADS__)
-#include "lf_POSIX_threads_support.h"  // (Not C++11 or later) or no threads support
+    #if __STDC_VERSION__ < 201112L || defined (__STDC_NO_THREADS__)
+// (Not C++11 or later) or no threads support
+#include "lf_POSIX_threads_support.c"
```

Comment:
I think we decided not to `#include` `.c` files.

Reply:
Jep. Old code, not adjusted to the new build config yet...